config/core/configmaps/features.yaml

```diff
@@ -50,9 +50,15 @@ data:
     # Indicates whether multi container support is enabled
     #
     # WARNING: Cannot safely be disabled once enabled.
-    # See: https://knative.dev/docs/serving/feature-flags/#multi-containers
+    # See: https://knative.dev/docs/serving/configuration/feature-flags/#multiple-containers
     multi-container: "enabled"
 
+    # Indicates whether multi container probing is enabled
+    #
+    # WARNING: Cannot safely be disabled once enabled.
+    # See: https://knative.dev/docs/serving/configuration/feature-flags/#multiple-container-probing
+    multi-container-probing: "disabled"
```

Comment:
I think we might not just want to enable this when `multi-container` is enabled, as it will change the QP spec (array instead of single probe) which causes a redeploy of all Knative Services, thus a new flag.

Reply:
Also it is a good idea for people to test this and then we should remove the flag once this is mature enough.

Reply:
Added docs in https://github.com/knative/docs/pull/5855